sunpy/map/mapbase.py

```diff
@@ -17,6 +17,7 @@
 from matplotlib import cm
 from matplotlib.backend_bases import FigureCanvasBase
 from matplotlib.figure import Figure
+from dask.array import Array as DaskArray
```

Comment:
I think this should probably be in the `_repr_html_` function, otherwise we will have to add dask to the main dependencies.

Reply:
The odds of using dask array somewhere else in mapbase in the future are high. I would just catch the import error here and set a flag, e.g.

```suggestion
DASK_INSTALLED = False
try:
    from dask.array import Array as DaskArray
    DASK_INSTALLED = True
except ImportError:
    pass
```